index.js

```diff
@@ -112,9 +176,6 @@ module.exports = {
   },
 
   _shouldIncludeFiles: function() {
-    if (process.env.EMBER_CLI_FASTBOOT) {
```

Comment:
I believe this was here so Mirage doesn't blow up when using fastboot. Does this no longer happen?